api-testing/functional/tests/package.json

```diff
@@ -2,15 +2,14 @@
   "scripts": {
     "testStudent": "cypress run --browser chrome --spec cypress/integration/01-student/*.spec.ts",
     "testInstitution": "cypress run --browser chrome --spec cypress/integration/02-institution/*.spec.ts",
-    "testMinistry": "cypress run --browser chrome --spec cypress/integration/03-ministry/*.spec.ts",
     "Run cypress headless mode command in student": "npm run testStudent",
     "Run cypress headless mode command in institution": "npm run testInstitution",
-    "Run cypress headless mode command in ministry": "npm run testMinistry",
     "Run cypress headless mode": "npx cypress run",
-    "Run cypress headed mode": "npx cypress open "
+    "Run cypress headed mode": " npx cypress open "
   },
   "devDependencies": {
-    "cypress": "^9.6.1",
+    "cypress": "^9.5.1",
```

Comment:
It seems that there was a downgrade from 9.6 to 9.5, is there any specific reason to this?

Reply:
Updated to version 9.7.0